File: ruby/7kyu/disemvowel.rb
str = "This website is for losers LOL!"

def disemvowel(str)
  vowels = "aAeEiIoOuU"
  news = ""
  str.each_char{ |c|
    if not vowels.include? c then 
      news += c
    end 
  }
  return news
end

puts disemvowel(str)

File: rust/8kyu/src/solutions/disemvowel.rs

fn disemvowel(s: &str) -> String {
    let vowels = "aeiouAEIOU";
    let out = s.chars().filter(|x| !vowels.contains(*x)).collect();
    out
}

fn main() {
    println!("{}", disemvowel("This website is for losers LOL!"));
}